security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -228,22 +228,13 @@ func (p *Plugin) constructFederatedTokenRequest(parameters security.StsRequestPa
 	}
 	req.Header.Set("Content-Type", contentType)
 	if pluginLog.DebugEnabled() {
-		dQuery := map[string]string{
-			"audience":           aud,
-			"grantType":          parameters.GrantType,
-			"requestedTokenType": tokenType,
-			"subjectTokenType":   parameters.SubjectTokenType,
-			"subjectToken":       "redacted",
-			"scope":              reqScope,
-		}
-		dJSONQuery, _ := json.Marshal(dQuery)
+		query["subjectToken"] = "redacted"
```

Comment:
nit: would be safer to copy the query map to another map and set the redacted field in that map